ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -658,15 +668,16 @@ double UncoupledProcessesTimeLoop::computeTimeStepping(
         }
         else
         {
-            if (std::abs(dt -prev_dt) < std::numeric_limits<double>::min()
-                && _last_step_rejected)
+            if (std::abs(dt - prev_dt) < std::numeric_limits<double>::min() &&
+                _last_step_rejected)
             {
-                OGS_FATAL("\tThis time step is rejected and the new computed"
-                          " step size is the same as\n"
-                          "\tthat was just used.\n"
-                          "\tSuggest to adjust the parameters of the time"
-                          " stepper or try other time stepper.\n"
-                          "\tThe program stops");
+                OGS_FATAL(
+                    "\tThis time step is rejected and the new computed"
+                    " step size is the same as\n"
+                    "\tthat was just used.\n"
+                    "\tSuggest to adjust the parameters of the time"
+                    " stepper or try other time stepper.\n"
+                    "\tThe program stops");
```

Comment:
I know it's not introduced in this PR, but OGS_FATAL here might prevent final cleanup, such as writing the last (failed) timestep result.